packages/core/src/types/ISurface.ts

```diff
@@ -0,0 +1,14 @@
+import Point3 from './Point3';
+
+export interface ISurface {
+  readonly id: string;
+  readonly sizeInBytes: number;
+  readonly frameOfReferenceUID: string;
+  getColor(): Point3;
+  setColor(color: Point3): void;
+  getPoints(): number[];
+  getPolys(): number[];
+  getSizeInBytes(): number;
+  setPoints(points: number[]): void;
+  setPolys(polys: number[]): void;
```

Comment:
Since we spell everything else out, could we use setPolygons?

Reply:
polys ans setPolys is vtk terminology https://kitware.github.io/vtk-js/api/Common_DataModel_PolyData.html#setPolys